apps/backoffice/src/app/api/auth/[...nextauth]/auth.ts

```diff
@@ -331,16 +341,15 @@ const toUser = ({
     id: identityTokenPayload.organization.id,
     name: identityTokenPayload.organization.name,
     role: identityTokenPayload.organization.roles[0]?.role,
-    logo_url: "url"
+    logo_url: "url" // TODO: retrieve institution logo from selfcare
```

Comment:
This TODO is left intentionally?

Reply:
@giamma1295 yes, i will manage it in a next dedicated PR 😉